src/styles/components/watch/watch-animations.scss

```diff
@@ -0,0 +1,5 @@
+@keyframes arrow-rotate-anim {
+  to {
+    transform: rotate($watch-steps-size * 1deg * 60);
```

Comment:
The rotation duration for the second and minute hands should be 60s and 5s respectively, but the animation is not configured for these durations.